intellij/src/nl/tudelft/watchdog/intellij/logic/InitializationManager.java

```diff
@@ -51,7 +53,13 @@
     private InitializationManager(Project project) {
         // Initialize persisters
         // Double getPath() because they are different methods on different objects
-        File baseFolder = new File(PluginManager.getPlugin(PluginId.findId("nl.tudelft.watchdog")).getPath().getPath());
+        final IdeaPluginDescriptor plugin = PluginManager.getPlugin(PluginId.findId("nl.tudelft.watchdog"));
+
+        if (plugin == null) {
+            throw new IllegalArgumentException("Plugin id \"nl.tudelft.watchdog\" could not be found in the list of installed plugins.");
```

Comment:
can we get the pluginid programmatically?

Reply:
This is specified in the `META-INF/plugin.xml` which is quite tedious to parse as XML. If we want to improve this, I would put it in a separate PR.